packages/components/src/components/Flyout/Flyout.tsx

```diff
@@ -93,9 +98,10 @@ export type FlyoutContentProps = {
 
 export const FlyoutContent = (
     {
-        align = 'start',
```

Comment:
why nor alphabetical

Reply:
to not separate `width` from `maxWidth` but don't have a strong preference